docs/Architectural-Overview.md

```diff
@@ -125,7 +130,166 @@ Its a REST service which provides APIs to stores build wise test results and agg
 ```
 
 # Hardware Configuration for test automation setup
-VMs or Physical machines with 16-32 GB RAM and 8-16 logical cores would be a decent configuration for a test machine. The deployment view [Test Execution Deployment](Test-Execution-Deployment-View.md) discusses more about the deployment structure. The least requirement is that there should be a network connectivity of these Cloud Form Vms or physical machines to the Cortx setups in labs.
+VMs or Physical machines with Linux flavor having 16-32 GB RAM and 8-16 logical cores would be a decent configuration for a test machine. Minimal test machine configuration is 4 GB RAM and 2 logical cores  which is suitable for development environment. The deployment view [Test Execution Deployment](Test-Execution-Deployment-View.md) discusses more about the deployment structure. The least requirement is that there should be a network connectivity of these Cloud Form Vms or physical machines to the Cortx setups in labs or public cloud services like AWS EC2 .
```

Comment:
IMO, We can remove the references to CloudForm VMs etc. specific to Seagate.

Reply:
I have added a generic VM.